WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -695,5 +695,7 @@ private extension InteractiveNotificationsManager {
     func disableWordPressNotificationsIfNeeded() {
         let notificationFilteringService = NotificationFilteringService()
         notificationFilteringService.disableWordPressNotificationsIfNeeded()
+
+        notificationFilteringService.cancelAllPendingWordPressLocalNotifications()
```

Comment:
```suggestion
```

This is called on `Jetpack` at the moment when notifications are enabled. Since it has a `.isWordPress` check inside the method, it won't get called.

Reply:
I've removed this thanks